docs/services.md

```diff
@@ -11,8 +11,20 @@ aliases:
 
 Services are built-in software packages that make it easier to add complex capabilities such as motion planning or object detection to your machine.
 
+{{< imgproc src="/viam/machine-components.png" alt="Machine components" resize="1000x" style="max-width:650px" >}}
+<br>
+
+Even though many services run locally within `viam-server`, you can think of them as discrete building blocks; you can run your machine using none, some, or all of them, depending on your use case.
+Configure just the services you want to use.
```

Comment:
The previous sentence covers that sufficiently I think
```suggestion
```

Reply:
I put that in here in an attempt to transition to the next paragraph about configuration, and to explain *how* someone would use some, none or all services seeing as they're already on their machine.

Reply:
Okay--I think we can make this more concise and the next paragraph covers it well enough beyond this:
```suggestion
To use a given service, add it to your machine's configuration.
```